src/page/MyShopPage/components/EventTable/index.tsx

```diff
@@ -20,9 +21,10 @@ export default function EventTable() {
   const [editMenu, setEditMenu] = useState(false);
   const [selectAll, setSelectAll] = useState(false);
   const [selectedEventIds, setSelectedEventIds] = useState<number[]>([]);
-  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isModifyErrorModalOpen, setIsModifyErrorModalOpen] = useState(false);
+  const [isDeleteErrorModalOpen, setIsDeleteErrorModalOpen] = useState(false);
   const [modalMessage, setModalMessage] = useState('');
-  const { mutate: deleteEvent } = useDeleteEvent(shopData!.id, selectedEventIds);
+  const { mutate: deleteEvents } = useDeleteEvent(shopData!.id, selectedEventIds);
```

Comment:
suspenseQuery를 적용했던 것 같은데, assertion이 없어져도 괜찮을까요~?
```suggestion
  const { mutate: deleteEvents } = useDeleteEvent(shopData.id, selectedEventIds);
```